src/utils/createTabNavigator.js

```diff
@@ -70,27 +70,23 @@ export default function createTabNavigator(TabView: React.ComponentType<*>) {
       return route.routeName;
     };
 
-    _handleOnTabPress = ({ route }) => {
+    _handleOnTabPress = ({ route, focused }) => {
```

Comment:
Can you change it back to using the old logic for `isFocused`? Bottom Navigation doesn't provide this property and I'm planning to remove it from react native tab view too.

Reply:
Unfortunately, the old logic erroneously returns `true` every time. I think the navigation state changes propagate before this logic is parsed.

Reply:
@ashoat maybe need to change it in react native tab view. let's do the change and I'll fix the behavior later today.